src/components/YourCompany/OfficeAddresses.vue

```diff
@@ -50,7 +50,7 @@
               </v-btn>
             </span>
             <span class="more-actions">
-              <v-menu offset-y>
+              <v-menu offset-y attach=".more-actions">
```

Comment:
This works perfectly in the browser but causes a warning in Jest (but still works). Its purpose it to attach the v-menu to its parent div (instead of the DOM root) so you can actually find it in the component under test.